pybase/region/client.py

```diff
@@ -142,7 +142,6 @@ def _send_request(self, rq, lock_timeout=10):
         to_send = pack(">IB", total_length - 4, len(serialized_header))
         to_send += serialized_header + rpc_length_bytes + serialized_rpc
 
-        pool_id = my_id % self.pool_size
         try:
             # todo: quick hack to patch a deadlock happening here. Needs revisiting.
             with acquire_timeout(self.write_lock_pool[pool_id], lock_timeout) as acquired:
```

Comment:
There's still a reference to the `write_lock_pool` here

Reply:
aaah, I'll get rid of that

Reply:
its gone